src/common/config_opts.h

```diff
@@ -863,7 +863,8 @@ OPTION(rbd_readahead_trigger_requests, OPT_INT, 10) // number of sequential requ
 OPTION(rbd_readahead_max_bytes, OPT_LONGLONG, 512 * 1024) // set to 0 to disable readahead
 OPTION(rbd_readahead_disable_after_bytes, OPT_LONGLONG, 50 * 1024 * 1024) // how many bytes are read in total before readahead is disabled
 OPTION(rbd_clone_copy_on_read, OPT_BOOL, false)
-OPTION(rbd_object_map, OPT_BOOL, false) // whether to enable the RBD object map
+OPTION(rbd_exclusive_lock, OPT_BOOL, false) // whether the rbd CLI should enable the RBD exclusive lock
+OPTION(rbd_object_map, OPT_BOOL, false) // whether the rbd CLI should enable the RBD object map
```

Comment:
hmm, looking at these again it'd be simpler if we used rbd_default_features, similar to how it uses the other rbd_default_* options. Then we could have rbd.cc-specific options for adding object_map + exclusive lock feature bits to the defaults, just like --image-shared removes them.